### ISSUE TITLE
UV4 does not uses Accept header for content negotiation when fetching v3 info.json

### ISSUE THREAD

**Issue by nicolasfranck** [UniversalViewer/universalviewer#951]
**UV version:**

```
 universalviewer@4.0.25
```

or I think. At least the one working now at https://universalview.dev

**I'm submitting a:**

- [x] bug report => please fork one of these [codesandbox examples](https://github.com/UniversalViewer/universalviewer/wiki/UV-Examples) with a repro of your issue and include a link to it below
- [ ] feature request => please use the [user stories repo](https://github.com/UniversalViewer/user-stories)
- [ ] support request => Please do not submit support requests here, use [stackoverflow](http://stackoverflow.com/questions/tagged/iiif)

**Current behavior:**

When referencing a v3 image service in my v3 manifest,
the info.json is not fetched with the appropriate `Accept`
header like documented [here](https://iiif.io/api/presentation/3.0/#63-responses).
So even when I set the service type to `ImageService3`,
it would still get the old v2 info.json (if that is default version
when no accept header is provided).

In other words: the UV4 does not support content negotiation for image api requests
like documented in the iiif documentation. 

I believe there
was some debate in the past within the IIIF community about wether
one should use content negotiation, or just use different urls.

Or am I missing something that would trigger content negotiation?
